operators/test/e2e/keystore_test.go

```diff
@@ -98,7 +99,96 @@ func TestUpdateSecureSettings(t *testing.T) {
 			},
 
 			// keystore should be updated accordingly
-			stack.CheckKeystoreEntries(k, s.Elasticsearch, nil),
+			stack.CheckESKeystoreEntries(k, s.Elasticsearch, nil),
+
+			// cleanup extra resources
+			helpers.TestStep{
+				Name: "Delete secure settings secret",
+				Test: func(t *testing.T) {
+					err := k.Client.Delete(&secureSettings)
+					require.NoError(t, err)
+				},
+			},
+		).
+		RunSequential(t)
+}
+
+func TestUpdateKibanaSecureSettings(t *testing.T) {
+	k := helpers.NewK8sClientOrFatal()
+
+	// user-provided secure settings secret
+	secureSettingsSecretName := "secure-settings-secret"
+	secureSettings := corev1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      secureSettingsSecretName,
+			Namespace: params.Namespace,
+		},
+		Data: map[string][]byte{
+			// this needs to be a valid configuration item, otherwise Kibana refuses to start
+			"logging.verbose": []byte("true"),
+		},
+	}
+
+	// set up a 1-node Kibana deployment with secure settings
+	s := stack.NewStackBuilder("test-keystore").
```

Comment:
It could be convenient to be able to differentiate this test from that for ES. So I would see well to name it `test-kb-keystore` and the other `test-es-keystore`?